DOCUMENTATION.md

```diff
@@ -99,6 +100,20 @@ Name | Type | Description | `flatpak override` equivalent
 Talks | Input | Allow the application to talk to session services. <br /> <br /> For example, adding `org.freedesktop.Notifications` will allow the application to send notifications. | `--talk-name=[NAME]`
 Owns | Input | Allow the application to own session services under the given name. | `--own-name=[NAME]`
 
+
+### Portals
+
+Portals, also know as dynamic permissions, are granted by users during runtime.
```

Comment:
```suggestion
[Portals](https://github.com/flatpak/flatpak/wiki/Portals), also known as dynamic permissions, are processes that run separately from the application to allocate data to the application only for the session.
```

I think the way it is right now is too vague and a bit hard to understand.

Reply:
Well, I think we will need to find a middle ground :laughing:  because now is way too technical, e.g. it really not necessary to explain `processes that run separately from the application`, What we need to communicate here is that these are permissions that do not come by default and that is up to the user to allow these accesses  or not.

Reply:
Another thing that we need to include there somehow, is that these take precedence over static permissions.

Reply:
> Another thing that we need to include there somehow, is that these take precedence over static permissions.

Can you elaborate on this? I edited my suggestion, but I failed to write the last sentence properly.

Reply:
We need to, somehow, explain that, if you revoke webcam access by removing `--devices=all` static permission, but later allow access though devices portal, the portals will take higher priority and allow the application to access the webcam.

This a pretty common confusion among Flatpak users, especially when talking about filesystem, e.g. hey I removed all filesystem permissions in Flatseal, but the application can still access this file I opened last week.

That happens because the access to that file was done through the document portals, and that takes higher priority over static permissions.

Reply:
I think I think I figured it out. What do you think?

Reply:
> allocate specific data to an application without needing to access a range of permissions

This part is still way too technical, we really need something for regular people like "Portals are not granted by default, and require users to explicitly allow these accesses from the application".

> For example, disabling all devices using static permissions (`--nodevice=all`) but enabling device access through portals will ignore the static permission as it prioritizes portals, thus giving the application access to devices.

This is pretty close, but I would use something more specific like the webcam to make it easier to grasp.

Reply:
Just updated. What do you think?